go_sequencer/common/l1tx.go

```diff
@@ -221,7 +221,7 @@ func L1UserTxFromBytes(b []byte) (*L1Tx, error) {
 	pkCompB := b[20:52]
 	pkCompL := SwapEndianness(pkCompB)
 	copy(tx.FromBJJ[:], pkCompL)
-	fromIdx, err := IdxFromBytes(b[52:58])
+	fromIdx, err := AccountIdxFromBytes(b[52:55])
```

Comment:
just to confirm, is it okay to leave the `b[55:58]` range empty?

Reply:
Yeah, Okay, and besides that, there are other gaps too.